src/vendor/highlight/_theme.scss

```
@@ -0,0 +1,85 @@
+@use '../../compiled/tokens/scss/color';
+@use '../../compiled/tokens/scss/opacity';
+@use '../../mixins/a11y';
+@use 'sass:color' as sasscolor;
+
+/// Cloud Four theme for Prism syntax highlighting.
+///
+/// @link https://highlightjs.readthedocs.io/en/latest/theme-guide.html
+
+.hljs,
+.hljs-subst {
+  color: color.$text-light;
+}
+
+.hljs-comment,
+.hljs-quote,
+.hljs-punctuation {
+  color: sasscolor.change(color.$text-light, $alpha: (opacity.$muted * 0.85));
+}
+
+.hljs-name,
+.hljs-built_in,
+.hljs-title,
+.language-scss .hljs-variable {
+  color: color.$base-fuchsia-lighter;
+}
+
+.hljs-keyword,
+.hljs-meta,
+.hljs-section {
+  color: color.$base-purple-lighter;
+}
+
+.hljs-attr,
+.hljs-params,
+.hljs-property,
+.hljs-selector-class,
+.hljs-template-variable {
+  color: color.$base-orange-lighter;
+}
+
+.hljs-literal,
+.hljs-selector-tag,
+.hljs-symbol,
+.hljs-type {
+  color: sasscolor.change(
+    sasscolor.mix(color.$base-fuchsia-lighter, color.$base-orange-light),
+    $saturation: sasscolor.saturation(color.$base-fuchsia-lighter)
+  );
+}
+
+.hljs-meta,
+.hljs-number,
+.hljs-selector-attr,
+.hljs-selector-pseudo {
+  color: sasscolor.change(
+    sasscolor.mix(color.$base-green-lighter, color.$base-blue-lighter),
+    $saturation: sasscolor.saturation(color.$base-green-lighter)
+  );
+}
+
+.hljs-string {
+  color: color.$base-green-lighter;
+}
+
+.hljs-attribute,
+.hljs-regexp,
+.hljs-template-tag,
+.hljs-variable {
+  color: color.$base-blue-lighter;
+}
+
+.hljs-selector-id,
+.hljs-title.class_ {
+  color: sasscolor.adjust(
+    sasscolor.mix(color.$base-orange-lighter, color.$base-green-lighter),
+    $saturation: +10%
+  );
+}
+
+/// Syntax-Highlighted Code Block Plugin Customizations
+
+.shcb-language {
+  @include a11y.sr-only;
+}
```

I had experimented with a more interesting visual display, which I showed in our last design team meeting, but in practice there were a _lot_ of bugs to resolve. I decided it was best to maintain the current appearance and to consider enhancements for a later date.

---

That makes sense. Can we add a bit of this info in a comment? Something along these lines?

```suggestion
/// Syntax-Highlighted Code Block Plugin Customizations

///
/// The plugin adds some content showing the language the code is written in.
/// This is helpful info, and we explored an interesting visual display,
/// but in practice, there were a lot of bugs to resolve.
///
/// For now, we hide it visually and expose it to screen readers.
///
.shcb-language {
  @include a11y.sr-only;
}
```